sssom/cli.py

```diff
@@ -172,7 +173,13 @@ def validate(input: str):
 
 @main.command()
 @input_argument
-@output_directory_option
+@click.option(
```

Comment:
why not instead at the `default=os.getcwd()` to the `@output_directory_option` above? should be useful for partition as well!

Reply:
Hence [the question in the ticket](https://github.com/mapping-commons/sssom-py/issues/230#issuecomment-1101780757).

Reply:
Done.

Reply:
lovely